src/components/SearchBar/SearchBar.tsx

```diff
@@ -27,24 +30,39 @@ interface ISearchBarDispatchProps {
   setSearchValue: (searchValue: string) => void;
 }
 
-export class SearchBarComponent extends React.Component<ISearchBarProps> {
+export class SearchBarComponent extends React.Component<ISearchBarProps, any> {

```

Comment:
Would be nice if you could provide an interface for the state. I would name it `ISearchBarState` with the searchValue you added in the state.